snapcraft/plugins/ruby.py

```diff
@@ -90,13 +97,19 @@ def __init__(self, name, options, project):
             raise SnapcraftEnvironmentError(
                 'Ruby version dir not in version_map.')
 
+    def snap_fileset(self):
+        fileset = super().snap_fileset()
+        fileset.append('-include/')
+        fileset.append('-share/')
+        return fileset
+
     def pull(self):
         super().pull()
         os.makedirs(self._ruby_part_dir, exist_ok=True)
         self._ruby_tar.download()
         self._ruby_install(builddir=self._ruby_part_dir)
         self._gem_install()
-        if self._install_bundler:
+        if self._use_bundler:
```

Comment:
Why not just evaluate `selfl.options.use_bundler` here instead of saving it off?